library/core/src/macros/mod.rs

```diff
@@ -735,8 +737,9 @@ macro_rules! unimplemented {
 /// ```
 #[macro_export]
 #[stable(feature = "todo_macro", since = "1.40.0")]
+#[allow_internal_unstable(core_panic)]
 macro_rules! todo {
-    () => ($crate::panic!("not yet implemented"));
+    () => ($crate::panicking::panic("not yet implemented"));
```

Comment:
It's unfortunate that this is necessary, considering that all users crates need to go through the `panic!` macro. Is there an open issue to track improving the compilation performance of `panic!`?

Reply:
The difference is very small, so I'm not sure it really matters in most cases. I suppose we could have `panic!()` expand to a direct call to panicking::panic() if there's no formatting placeholders.